tests/tt_metal/tt_metal/unit_tests/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+
+set(UNIT_TESTS_SRC
```

Comment:
Did you consider using `GLOB_RECURSE`?
It has a cost and it is less important for those switching from `make`, but it is convenient to not worry about adding a new file to the CMakeLists